app/include/zmk/hid.h

```diff
@@ -85,7 +86,7 @@ static const u8_t zmk_hid_report_desc[] = {
     USAGE_GEN_DESKTOP_KEYPAD,
     /* REPORT_SIZE (8) */
     HID_GI_REPORT_SIZE,
-    0x02,
+    0x01,
```

Comment:
In doing some further testing on this, when using my MDK kit over USB, I do *not* get key events with this change and the change to the `ZMK_HID_MAX_KEYCODE` value. My *guess* is that this is related to the USB data being too large after this change, and causing problems for the onboard USB HUB, perhaps?

@innovaker Thoughts on reverting just these changes, in favor of switching to a `6KRO` (and future `#KRO`) type HID report/descriptor for this in a future PR?

Doing so would of course limit the actual *use* of some of these new HID keycodes defined here, but allow us to move forward with other changes while we work on iterating on the actual transmitted HID report encoding/format.

Reply:
Casual observers, this was discussed throughout the day on Discord.  We replaced a core part of the system in an effort to fix this issue.  More delays follow below.